lib/smarterEncryption.js

```diff
@@ -130,5 +130,57 @@ function generateSmarterEncryptionRuleset (domains, startingRuleId = 1) {
     return rules
 }
 
+/**
+ * Create a rule to disable or enable automatic HTTPS upgrades for a set of domains.
+ * @param {string[]} domains
+ * @param {number | undefined} id (optional) rule ID
+ * @param {'allow' | 'upgrade'} type 'allow' to create an allowlist rule, 'upgrade' to create an upgrade rule
```

Comment:
Just checking, do you mean "allow" here instead of "ignore"? I ask since declarativeNetRequest rules use "allow", but our tracker blocking rules use "ignore". I wasn't sure what was correct for the https feature configuration.

Reply:
I'm sticking with DNR terminology here, i.e. "allow". I'm not sure that there is a reason for SE to use blocklist terms.

Reply:
OK, fair enough. Just wanted to check that wasn't a typo.